### ISSUE TITLE
Distributed MNIST example fails to run

### ISSUE THREAD

**Issue by johnugeorge** [kubeflow/pytorch-operator#35]
With the latest pytorch, Dataloader argument batch size has to be an integer.   Also, usage of indexing 0-dim Tensor is deprecated (See https://github.com/pytorch/pytorch/issues/6061) .
